tests/dags/providers/provider_api_scripts/test_stocksnap.py

```diff
@@ -17,42 +19,131 @@
 )
 
 
-def test_get_image_pages_returns_correctly_with_none_json():
+stocksnap = StockSnapDataIngester()
+image_store = ImageStore(provider=prov.STOCKSNAP_DEFAULT_PROVIDER)
+stocksnap.media_stores = {"image": image_store}
+
+
+#     def _get_image_info(item):
+#     def _get_title(item):
+#     def _get_metadata(item):
+# def ingest_records(self, **kwargs) -> None:
+# def get_batch(self, query_params: Dict) -> Tuple[Optional[List], bool]:
+# def get_should_continue(self, response_json):
+# def get_batch_data(self, response_json):
+# def process_batch(self, media_batch):
```

Comment:
Looks like this was accidentally left in :)

Reply:
Ugh, yes, thank you!